hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieTimeline.java

```diff
@@ -126,6 +129,13 @@
    */
   HoodieTimeline getCommitsAndCompactionTimeline();
 
+  /**
+   * Timeline to just include replace instants that have valid (commit/deltacommit) actions.
+   *
+   * @return
+   */
+  HoodieTimeline getCompletedAndReplaceTimeline();
```

Comment:
May be you have used this API in a different PR, but where exactly are we going to use this specific method ?

Reply:
yes, this is used to build filesystem view in #1859, can move this to that PR